css/header.css

```diff
@@ -0,0 +1,163 @@
+.header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	background: var(--bg-color-1);
+	box-shadow: var(--box-shadow);
+	padding: 1.5rem 7%;
+	z-index: 1000;
+	display: flex;
```

Comment:
such high values of z-index isn't required

Reply:
Got it thanks for the suggestion!